Added governments and sales files

warp-core opened the pull request:
Added sales file and cleared all the shipyards and outfitters.
Added governments file and cleared all the governments we definitely won't be using.

warp-core committed:
Clearing trading and fleets and making uninhabited

Cleared all systems up to and not including Alkaid

warp-core committed:
Cleared shipyards and outfitters

warp-core committed:
Revert "Clearing trading and fleets and making uninhabited"

This reverts commit efd1b522030cf2d75657c608c8fd46e1c775f629.

warp-core committed:
Cleared governments we definitely won't be using

warp-core committed:
Updated governments.txt

Moved cleared governments to the bottom.
Modified player reputation with 'deep' government.
Added 'Paradise Planets' government.

warp-core committed:
Added start file

warp-core commented:
I hope the 'clear' keyword works the way I think it does...

warp-core committed:
Changed start location to Luna for testing purposes

Makuta-Miras commented:
Take out the alien governments, clear the start for now, and clear all of the shipyards and outfitters too.

warp-core committed:
Updated governments and sales.txt's

Cleared all alien governments except Quarg and Pug.
Cleared all shipyards and outfitters completely.

warp-core committed:
Cleared start

warp-core committed:
Added copyright information to all files

Makuta-Miras commented:
Looks good now; I think depending on how the story ends up being, the player might make more sense being either Independent or aligned to a corporation (In which case we'll want to change their starting swizzle) but looks good for now. Merged!
